src/pybind/mgr/dashboard/controllers/auth.py

```diff
@@ -1,15 +1,26 @@
 # -*- coding: utf-8 -*-
 from __future__ import absolute_import
 
+import http.cookies
 import logging
+import sys
 
+try:
+    from urlparse import urlparse
+except ImportError:
+    from urllib.parse import urlparse
```

Comment:
This is trying first the Python 2 syntax, which is the deprecated version. If we want to make this py2 backportable, I think it's better to use [`six urllibparse`](https://six.readthedocs.io/#module-six.moves.urllib.parse), but I'm not sure if we should introduce Py2-3 compatible code in `master`.

What do you thinkg @tchaikov ?

Reply:
There is no need for using it any more. I have changed the way of knowing if it's 'https' scheme as suggested by you and with that we no more need to use this piece of code!

Reply:
Perfect, then. Thanks @avanthakkar !